virtualpdu/tests/integration/test_entrypoint.py

```diff
@@ -56,25 +61,77 @@ def test_config(self):
+                    self._get_entry_point_path('virtualpdu'),
+                    f.name,
+                ])
+
+                self.turn_off_outlet(community='public',
+                                     listen_address='127.0.0.1',
+                                     outlet=5,
+                                     port=9998)
+
+                self.turn_off_outlet(community='public',
+                                     listen_address='127.0.0.1',
+                                     outlet=2,
+                                     port=9997)
+            finally:
+                # NOTE(mmitchell): The process shouldn't have died on it's
+                #                  own. Kill it.
+                p.kill()
+
+    def test_sigint_stops_the_process(self):
+        with tempfile.NamedTemporaryFile() as f:
+            f.write(bytearray(TEST_CONFIG, encoding='utf-8'))
+            f.flush()
+            try:
+                p = subprocess.Popen([
+                    sys.executable,
+                    self._get_entry_point_path('virtualpdu'),
+                    f.name,
+                ])
+
+                self.assertIsNone(p.poll())
+                p.send_signal(signal.SIGINT)
+                time.sleep(0.1)  # NOTE(mmitchell): Give it 100ms to stop.
+                self.assertIsNotNone(p.poll())
+            finally:
+                try:
+                    p.kill()
+                except OSError:
+                    pass
+
+    def test_pdu_names_ips_and_ports_are_shown_on_stderr(self):
+        with tempfile.NamedTemporaryFile() as f:
+            f.write(bytearray(TEST_CONFIG, encoding='utf-8'))
+            f.flush()
+            try:
+                p = subprocess.Popen([
+                    sys.executable,
+                    self._get_entry_point_path('virtualpdu'),
+                    f.name,
+                ], stderr=subprocess.PIPE)
 
-            self.turn_off_outlet(community='public',
-                                 listen_address='127.0.0.1',
-                                 outlet=5,
-                                 port=9998)
+                t = threading.Timer(1, p.send_signal, [signal.SIGINT])
+                t.start()
+                stdout, stderr = p.communicate()
+                t.join()
 
-            self.turn_off_outlet(community='public',
-                                 listen_address='127.0.0.1',
-                                 outlet=2,
-                                 port=9997)
+                self.assertIn(b'my_pdu', stderr)
+                self.assertIn(b'127.0.0.1:9998', stderr)
 
-            p.kill()
+                self.assertIn(b'my_second_pdu', stderr)
+                self.assertIn(b'127.0.0.1:9997', stderr)
+            finally:
+                try:
+                    p.kill()
+                except OSError:
+                    pass
 
-    def turn_off_outlet(self, community, listen_address, outlet, port):
+    @staticmethod
```

Comment:
Instead you could drop the method at module level with a _ prefix and invoke it without a self, you save bytes and a line and you don't awkwardly invoke a CLASS method with SELF